apps/src/templates/Notification.jsx

```diff
@@ -35,12 +37,14 @@ const styles = {
   details: {
     fontFamily: '"Gotham 4r", sans-serif',
     fontSize: 14,
-    lineHeight: 2.5,
-    marginBottom: 16,
+    lineHeight: 1.5,
+    paddingTop: 6,
+    paddingBottom: 6,
```

Comment:
Reduced lineHeight because the gaps between multiple lines of detail text looked too big. Instead using paddingTop to ensure there's enough space between notice and details.